tests/appsec/test_blocking_addresses.py

```diff
@@ -54,6 +57,7 @@ def test_blocking(self):
     def setup_blocking_before(self):
         self.block_req2 = weblog.get("/tag_value/tainted_value_6512/200", headers={"X-Forwarded-For": "1.1.1.1"})
 
+    @irrelevant(context.weblog_variant == "nginx", reason="Tag adding happens before WAF run")
```

Comment:
out of curiosity, it means that it'll never be supported ?

Reply:
I'm not sure about never, but I can't see other use cases to set tags only after and conditionally on the blocking of response, except doing that just for having these tests passing. If there is a compelling, non purely test related use case, we can look at the viability.